hera_pspec/utils.py

```diff
@@ -70,6 +72,135 @@ def get_delays(freqs):
     delay = np.fft.fftshift(np.fft.fftfreq(freqs.size, d=np.median(np.diff(freqs))))
     return delay
 
+
+def spw_range_from_freqs(data, freq_range, bounds_error=True):
+    """
+    Return the first and last frequency array indices for a spectral window, 
```

Comment:
I think the name of the function is fine, but this line is a little confusing. The `first and last frequency array indices` sounds like I'm getting a frequency index, when really I'm getting a spectral window. For example, I can't do `freq_array[output]` with the output of this function. Perhaps 'Return the spectral window of the first and the last frequency specified in `freq_range`'

Reply:
Or perhaps "Return the spectral window that each frequency specified in `freq_range` falls in"

Reply:
OK, as long as the user understands "spectral window" to mean "a tuple that specifies the start and end indices of the frequency array", I think that's fine.